### DISCUSSION TITLE
Sorting Question

### DISCUSSION THREAD

**Discussion by Site3op** [UltimaHoarder/UltimaScraper#1510] [Q&A]
Is there a way I can put these into folders outside of the OnlyFans scrapper folder and not have the scrapper continuously keep downloading them once I remove them from said model folder. I would basically like to know if I can have it only scrape content once and not re-scrape the same content if I move it from the folders but also be able to scrape new content that wasn't scrapped previously. If this is possible I would greatly appreciate a method on how to get it to work. Thank you.

**Comment by LunarPenguin** [comment]
yes it can and is default behavior. make sure `overwrite_files` in `config.json` is set to `false`
